source/__tests__/integration/new-profile.test.js

```diff
@@ -0,0 +1,64 @@
+describe('Adding and removing a new Profile', () => {
+
+    beforeAll(async () => {
+        await page.goto('http://localhost:8000/#/profile/create')
+    })
+
+    it('displays a new profile span', async () => {
+        const span = await page.$eval('span[id="profileBanner"] > span', e => e.getAttribute('popover-title'))
+        await expect(span).toMatch(/Profile ID: Undefined/)
+    })
+
+    describe('Adding a Resource Template', () => {
+
+        beforeAll(async () => {
+            await page.waitForSelector('#addResource')
+            await page.click('#addResource')
+        })
+
+        it('displays a new resource template span', async () => {
+            const title = await page.$eval('span[id="0"] > span', e => e.getAttribute('popover-title'))
+            const span = await page.$eval('span[id="0"] > span', e => e.textContent)
+            expect(title).toMatch(/Resource ID: Undefined/)
+            expect(span).toMatch(/Resource Template/)
+        })
+
+        it('has six input fields for the profile admin data', async () => {
```

Comment:
my only question is whether checking the number of input fields is an adequate test.  I'm not sure what would be better - maybe looking for a specific label for a field to show it's really the right form?

Reply:
Ok, I added checks for the field labels.